fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsEnumerations.java

```diff
@@ -200,6 +199,7 @@ public static SavingsAccountTransactionEnumData transactionType(final SavingsAcc
                 optionData = new SavingsAccountTransactionEnumData(SavingsAccountTransactionType.AMOUNT_RELEASE.getValue().longValue(),
                         SavingsAccountTransactionType.AMOUNT_RELEASE.getCode(), "Release Amount");
                 break;
+            default:
```

Comment:
@percyashu @awasum I disagree with this, this seems wrong to me - I've raised new #997 for it.

PS: I'm happy to help code review such PRs, if you would like me to.

Reply:
@vorburger . That is Ok with me to review such PRs

I am looking into the points you raised wrt this merged PR.